### PR TITLE
Add logging documentation

### PR DESCRIPTION
**Pull request by leightonthomas**

### CODE REVIEW

File: logging.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
+# Info
+A 'production level' debug - less verbose version of debug
+
+• A member is being synced
+• A product availability is bei
```
Comment (kane-menicou) [inline logging.md:5]: Sentence needs finishing

File: logging.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
+# Info
```
Comment (kane-menicou) [inline logging.md:1]: Could we add information about `debug` also

Comment (leightonthomas) [reply logging.md:7]: Sure, added next commit

File: logging.md (hunk 1 above)
Comment (leightonthomas) [reply logging.md]: Fixed in next commit

File: logging.md (hunk 2 above)
Comment (kieljohn) [reply logging.md:7]: Happy with this resolution

File: logging.md (hunk 1 above)
Comment (kieljohn) [reply logging.md]: Sorted

**Pull request merged**